### ISSUE TITLE
How do I feed three inputs into tflearn's input_data()

### ISSUE THREAD

**Issue by Phillyclause89** [tflearn/tflearn#1128]
I made a reddit post of my [issue](https://www.reddit.com/r/learnmachinelearning/comments/bmdvgh/how_do_i_feed_three_inputs_into_tflearns_input/) since I can do a bit more with their markdown than I can with github's.  Long story short, I want my model (based off of Sentdex's pygta's use of inception_v3) to look at two images and a feature that could be one of nine possible values.  What's the best way to go about doing this?

**Comment by Hemantr05** [comment]
Hi @Phillyclause89 could you please elaborate?
And if you don't mind me asking, why would you require two images?

**Comment by Phillyclause89** [comment]
Oh golly I opened this along time ago.  Long story short I was trying to make a self driving car in GTA V using some YouTube tutorial. The tutorial was training the algorithm to make a steering input into the game based off of the current frame. I was curious if the model might be improved if I could feed multiple frames into it to give a sense of speed (I.e the faster the car is going the greater the pixel  difference would be for the current frame from the last.) I ended up just doing some preprocessing of my frames to merger the last 4 frames into a single four panel image and training off of that. It wasn’t very successful, but I learned a lot of python during that project.
